.circleci/config.yml

```diff
@@ -101,6 +101,8 @@ jobs:
       redis_tag: buster
     steps:
       - checkout
+      - restore_cache:
+          key: dependency-cache-{{ checksum "package-lock.json" }}
```

Comment:
doesn't the attach workspace on line 106 also attach the node_modules that was persisted on line 50?
Is this needed?

If it is needed, does this mean we can drop the restore cache on line 125?